java-generic-mqtt/src/main/java/com/cumulocity/generic/mqtt/client/GenericMqttSubscriber.java

```diff
@@ -0,0 +1,18 @@
+package com.cumulocity.generic.mqtt.client;
+
+/**
+ * Interface for subscribing to messages from the server.
+ * <p>
+ * <code>GenericMqttSubscriber</code> instances are setup using {@link GenericMqttClient#buildSubscriber(GenericMqttConfig)}
+ */
+public interface GenericMqttSubscriber extends AutoCloseable {
+
+    /**
```

Comment:
nitpick: sorry for late comment. We have two links to same class and that class will be linked as a parameter by javadoc tool. I think both of these should be \<code\> tags

Reply:
Done.